tests/test_xml_rules.py

```diff
@@ -21,12 +20,20 @@
 XML_FILENAME_PATH = os.path.join(os.path.dirname(__file__), '..',
                                  RESOURCE_FOLDER, XML_FILENAME)
 
+def dict_contains_subset(A, B):
+    """extract Dict A From B
+    DeprecationWarning: assertDictContainsSubset is deprecated for python 3.5
+    self.assertDictContainsSubset(expected, actual[0])"""
+    return dict([(k, B[k]) for k in A.keys() if k in B.keys()])
+
 
 class TestXMLtoDict(unittest.TestCase):
     "parse xml from sdk to python dict"
+    maxDiff = None
+
     def test_xml_to_dict(self):
```

Comment:
This test is going to fail the first time when validate.xml is changed as a part of SDK changes.
Please, use test version of the xml to be parsed (it can be shorter as well).